src/NewsletterBundle/objects/WebModules/MTPkgNewsletterSignoutCore/MTPkgNewsletterSignoutCore.class.php

```diff
@@ -114,6 +129,10 @@ protected function UnsubscribeUser($sPkgNewsletterUserMail = null, $sPkgNewslett
 
         if (!$bError) {
             $bConfirme = false;
+            /**
+             * @psalm-suppress UndefinedPropertyFetch
+             * @FIXME Does `fieldUseDoubleOptOut` exist?
+             */
             if (!$this->oModuleConfig->fieldUseDoubleOptOut || $this->IsUnsubscribeCodeValid($sUnsubscribeCode, $oNewsUser->id, $sPkgNewsletterGroupId)) {
```

Comment:
I wasn't sure if this field actually exists - I couldn't find it. Maybe it does not exist in all environments? If so, we should probably check for its existance